lerna.json

```diff
@@ -2,10 +2,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.0.0",
-  "command": {
-    "bootstrap": {
-      "ignore": "ethereumjs-block"
-    }
-  }
```

Comment:
ethereumjs-block is now integrated to Lerna workflow, so this change removes it from the ignore list for the bootstrap command.